akka-http/src/main/scala/akka/http/javadsl/server/Route.scala

```diff
@@ -45,17 +45,21 @@ trait Route {
   @InternalApi
   private[http] def delegate: scaladsl.server.Route
 
-  def flow(system: ActorSystem, materializer: Materializer): Flow[HttpRequest, HttpResponse, NotUsed]
+  def flow(
+    system:       ActorSystem,
+    materializer: Materializer): Flow[HttpRequest, HttpResponse, NotUsed]
 
   /**
    * Seals a route by wrapping it with default exception handling and rejection conversion.
    *
    * A sealed route has these properties:
+   *
    *  - The result of the route will always be a complete response, i.e. the result of the future is a
    *    `Success(RouteResult.Complete(response))`, never a failed future and never a rejected route. These
    *    will be already be handled using the implicitly given [[RejectionHandler]] and [[ExceptionHandler]] (or
    *    the default handlers if none are given or can be found implicitly).
    *  - Consequently, no route alternatives will be tried that were combined with this route.
+   *
```

Comment:
These extra blanklines don't seem to help rendering the text as a `<ul>` so I'll revert the changes in this file.